tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,10 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
+    assert len(vendor.inventory) == 3
+    assert item not in vendor.inventory
+    assert result is False
```

Comment:
👍 These checks the specified result, confirms the size of the inventory was unchanged, and that the invalid item wasn't somehow added to the list.

From one perspective, this might be too pessimistic. After all, why would trying to remove a non-existent value from a collection result in it being added. From another perspective, this might be too optimistic. Maybe we should check that the original three values are unchanged (not just that there are still three values at all). Tests can check a variety of conditions, and while not every test needs to check every condition, it's not a bad idea to have an occasional test that's a bit more suspicious of the results.